content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -243,23 +243,30 @@ The preceding configuration file defines a new context named `dev-ramp-up`.
 
 See whether you have an environment variable named `KUBECONFIG`. If so, save the
 current value of your `KUBECONFIG` environment variable, so you can restore it later.
-For example, on Linux:
-
+For example:
+Linux
 ```shell
 export  KUBECONFIG_SAVED=$KUBECONFIG
 ```
-
+Windows PowerShell
+```shell
+ $Env:KUBECONFIG_SAVED=$ENV:KUBECONFIG
+ ```
  The `KUBECONFIG` environment variable is a list of paths to configuration files. The list is
 colon-delimited for Linux and Mac, and semicolon-delimited for Windows. If you have
 a `KUBECONFIG` environment variable, familiarize yourself with the configuration files
 in the list.
 
-Temporarily append two paths to your `KUBECONFIG` environment variable. For example, on Linux:
+Temporarily append two paths to your `KUBECONFIG` environment variable. For example:<br>
 
+Linux:
 ```shell
 export  KUBECONFIG=$KUBECONFIG:config-demo:config-demo-2
 ```
-
+Windows PowerShell
+```shell
+$Env:KUBECONFIG_SAVED=(config-demo;config-demo-2)
```

Comment:
- Why do you need the raw line break?
- You could add two sub-headers, such as:
   ### Linux, using bash:
  ```shell
  some linux commands ...
  ```
  ### Windows Powershell:
  ```shell
  some windows commands ...
  ```
- Is this environment variable KUBECONFIG_SAVED correct?